analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ReturnEmptyCollectionInsteadOfNull.cs

```diff
@@ -17,7 +17,7 @@ IEnumerable<string> Strings2
             get
             {
                 return null; // Noncompliant
-//              ^^^^^^^^^^^^
+//                     ^^^^
             }
         }
```

Comment:
Please add a test case that would produce a secondary location. Like returning null on multiple branches.